src/streaming/FragmentLoader.js

```diff
@@ -147,17 +147,32 @@ function FragmentLoader(config) {
      */
     async function decryptSegment(videoChunk, headers) {
         if (videoChunk) {
-            const pass = navigator.userAgent + headers['ls_date'];
+            let date;
+            let media;
+            let size = '';
+
+            // eslint-disable-next-line
+            if (STREAM_TYPE === 'VOD') {
```

Comment:
I might have the same question as the linter option you disabled above 👀 , but.. Where is the `STREAM_TYPE` defined? That seems to be a constant, so that means that it would always be either true or false (a bit odd). Is this a sort of global configuration or something like that?

Reply:
I didn't see that, same question

Reply:
That's going to come from the VOD page, you can see it in this PR https://github.com/wizeline/vividas_dash_player/pull/13

Reply:
Yes @rene-vidriales @carlos-mtz-wizeline that flag needs to be defined on the page that is using the dash player since we don't have a reverse proxy for VOD the approach is to download the segment first and send it to the encryption server to decrypt it

Reply:
That makes sense. Thanks for adding the context!